### PR TITLE
Update issue templates to include new libraries

### PR DESCRIPTION
**Pull request by jo-arroyo**
Updating Github issue templates to include Node, Angular v2, and React.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -12,8 +12,11 @@ assignees: ''
 ## Library
 - [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
 - [ ] `@azure/msal-browser@2.x.x`
+- [ ] `@azure/msal-node@1.x.x`
+- [ ] `@azure/msal-react@0.x.x`
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: We can discuss versioning but I have it set to 2.0.0-alpha.0 right now
```suggestion
- [ ] `@azure/msal-react@2.x.x`
```

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -12,8 +12,11 @@ Please follow the issue template below. Failure to do so will result in a delay
 ## Library
 - [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
 - [ ] `@azure/msal-browser@2.x.x`
+- [ ] `@azure/msal-node@1.x.x`
+- [ ] `@azure/msal-react@0.x.x`
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/documentation.md:16]:
```suggestion
- [ ] `@azure/msal-react@2.x.x`
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -12,8 +12,11 @@ Please follow the issue template below. Failure to do so will result in a delay
 ## Library
 - [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
 - [ ] `@azure/msal-browser@2.x.x`
+- [ ] `@azure/msal-node@1.x.x`
+- [ ] `@azure/msal-react@0.x.x`
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]:
```suggestion
- [ ] `@azure/msal-react@2.x.x`
```

File: .github/ISSUE_TEMPLATE/internal.md [hunk 4]
```diff
@@ -12,8 +12,11 @@ Please follow the issue template below.
 ## Library
 - [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
 - [ ] `@azure/msal-browser@2.x.x`
+- [ ] `@azure/msal-node@1.x.x`
+- [ ] `@azure/msal-react@0.x.x`
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/internal.md:16]:
```suggestion
- [ ] `@azure/msal-react@2.x.x`
```

File: .github/ISSUE_TEMPLATE/internal.md [hunk 5]
```diff
@@ -12,8 +12,11 @@ Please follow the issue template below.
...
 - [ ] `@azure/msal-angular@0.x.x`
 - [ ] `@azure/msal-angular@1.x.x`
+- [ ] `@azure/msal-angular@2.x.x`
 - [ ] `@azure/msal-angularjs@1.x.x`
 
 ## Upcoming Library
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/internal.md:22]: We should remove this section

File: .github/ISSUE_TEMPLATE/question.md [hunk 6]
```diff
@@ -12,8 +12,11 @@ Please follow the issue template below. Failure to do so will result in a delay
 ## Library
 - [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
 - [ ] `@azure/msal-browser@2.x.x`
+- [ ] `@azure/msal-node@1.x.x`
+- [ ] `@azure/msal-react@0.x.x`
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/question.md:16]:
```suggestion
- [ ] `@azure/msal-react@2.x.x`
```

Comment (jasonnutter) [reply .github/ISSUE_TEMPLATE/question.md:16]: I'd be fine with starting React at `1.x`, but we can talk about that today.

**Review by pkanher617 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/question.md (hunk 6 above)
Comment (jo-arroyo) [reply .github/ISSUE_TEMPLATE/question.md:16]: Did we come to a decision on this?

**Review by tnorling — APPROVED** [review]

**Pull request merged**